packages/cli/test/unit/util/build/import-builders.test.ts

```diff
@@ -105,6 +105,7 @@ describe('importBuilders()', () => {
       await expect(client.stderr).toOutput(
         '> Installing Builder: @vercel/node'
       );
+      await expect(client.stderr).not.toOutput('npm WARN deprecated');
```

Comment:
I'd prefer not to use `.not.toOutput()` if possible, as it relies on a few seconds timeout and will make the tests start to take long as these add up. Not saying we can't use it here, but something to keep in mind.

Reply:
@TooTallNate The thing is, we should really be testing to make sure we don't print warnings when we don't intend to. If you can think of another way to test for that, I'm all ears. But I don't think its a good idea to remove these assertions.

Reply:
You may have to resort to adding a listener to `client.stderr` and manually append the chunks into a buffer (like this https://stackoverflow.com/a/49428486) that can be asserted after the command has finished.

Reply:
Let's keep it. Let's just be mindful when adding these.